custom_components/pyscript/__init__.py

```diff
@@ -217,9 +217,16 @@ async def state_changed(event):
             "value": new_val,
             "old_value": old_val,
```

Comment:
With f7169ac, `"value"` can be `StateVar(event.data.get('new_state'))`, or `None` if `new_state` is not provided (eg, when state variable is deleted.  Same for `"old_value"`.